src/modules/Accordion/Accordion.js

```diff
@@ -84,22 +94,39 @@ export default class Accordion extends Component {
     // The default prop should always win on first render.
     // This default check should then be removed.
     if (typeof this.props.defaultActiveIndex === 'undefined') {
-      this.trySetState({ activeIndex: -1 })
+      this.props.exclusive ? this.trySetState({ activeIndex: -1 }) : this.trySetState({ activeIndex: [-1] })
     }
+
+    //TODO: Can not accept an activeIndex and be inclusive at the same time
+    // if(this.props.exclusive == false && _.isNumber(this.props.activeIndex)){
+    //   this.trySetState({ activeIndex: [ this.props.activeIndex ] })
+    // }
```

Comment:
Correct, this is an `AutoControlledComponent` which means, if the user passes a prop then they are in control of the state but when there is no prop the Component is in control of the state.

`this.trySetState` will only set state if there is no prop defined.  In this regard, the component "auto controls" state for the user but also backs off if the user controls it with props.

It is up to the user to pass the correct combination of `activeIndex` and `exclusive`.   The alternative is to pass `defaultActiveIndex` which is only applied on the first render, then the component can update the activeIndex on subsequent renders.